src/templates/chat/roll-result.html

```diff
@@ -8,13 +9,28 @@
             <div class="chat-img" style="background-image:url('{{data.actor.img}}')"></div>
             {{/if}}
         </div>
+        <div class="card-content">
+            {{{data.description}}}
+        </div>
         <div class="blindable" data-blind="{{data.roll.blindroll}}">
             {{#if result.details}}<div class="chat-details">{{{result.details}}}</div>{{/if}}
             {{#if result.isFailure}}<div class='roll-result roll-fail'><b>{{localize 'OSE.Failure'}}</b> ({{result.target}})
             </div>{{/if}}
             {{#if result.isSuccess}}<div class='roll-result roll-success'><b>{{localize 'OSE.Success'}}</b>
                 ({{result.target}})</div>{{/if}}
             {{#if rollOSE}}<div>{{{rollOSE}}}</div>{{/if}}
+            {{#if data.save}}
```

Comment:
Does this come from `rollData.roll.save` or `rollData.save`?

Reply:
This comes from RollData.save, I've removed rollData.roll.save as it is not needed.